apps/package.json

```diff
@@ -12,7 +12,8 @@
     "test": "grunt test",
     "clean": "grunt clean",
     "build": "MOOC_DEV=1 grunt build",
-    "build:dist": "MOOC_DEV=0 grunt clean build"
+    "build:dist": "MOOC_DEV=0 grunt clean build",
+    "start": "MOOC_DEV=1 grunt dev"
```

Comment:
That must be the conflict

On Fri, Feb 19, 2016 at 1:02 PM, Brad Buchanan <notifications@github.com>
wrote:

> In apps/package.json
> <https://github.com/code-dot-org/code-dot-org/pull/6881#discussion_r53517627>
> :
>
> > @@ -12,7 +12,8 @@
> >      "test": "grunt test",
> >      "clean": "grunt clean",
> >      "build": "MOOC_DEV=1 grunt build",
> > -    "build:dist": "MOOC_DEV=0 grunt clean build"
> > +    "build:dist": "MOOC_DEV=0 grunt clean build",
> > +    "start": "MOOC_DEV=1 grunt dev"
>
> Didn't I already do this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6881/files#r53517627>.
>